src/Microsoft.AspNet.Server.Kestrel/Networking/Libuv.cs

```diff
@@ -216,6 +227,40 @@ public void tcp_bind(UvTcpHandle handle, ref SockAddr addr, int flags)
         {
             handle.Validate();
             Check(_uv_tcp_bind(handle, ref addr, flags));
+            if (PlatformApis.IsWindows)
+            {
+                tcp_bind_windows_extras(handle);
```

Comment:
@davidfowl I spent a while trying to get stuff to run on 2008 R2 SP1 but dnvm; wasn't completely happy and it had troubles with nuget; and I couldn't run build - so not sure as to the effects to the patch.